docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -590,30 +592,32 @@ After KYC is collected, you should be ready to receive the off-chain deposit. Wh
 
 </CodeExample>
 
-Amount should be collected in the interactive flow. However, it can also be specified by wallet as part of the request (see `amount_expected` of `GET /transaction/:id`). Please note, that you **must** specify amount fields, as wallet do not provide this information.
+The amount should be collected in the interactive flow. However, it can also be specified by the wallet as part of the request (see `amount_expected` of `GET /transaction/:id`).
+
+Please note that you **must** specify amount fields, as wallets do not provide this information.
 
-- `amount_in` is amount user requested. If `amount_expected` is not null, `amount_in` should be equal to `amount_expected`. In that case, amount should not be collected in the interactive flow.
-- `amount_out` is amount user will receive
-- `amount_fee` is total fees collected by the application.
-- `asset` part of amount fields is an [SEP-38][sep-38] formatted asset of the amount. In this example, it's set to USD, assuming user made a bank transfer to the system using USD.
+- `amount_in` is the amount the user requested. If `amount_expected` is not null, `amount_in` should be equal to `amount_expected`. In that case, the amount should not be collected in the interactive flow.
+- `amount_out` is the amount the user will receive.
+- `amount_fee` is the total fees collected by the application.
+- `asset` is part of the amount fields is a [SEP-38][sep-38] formatted asset of the amount. In this example, it's set to USD, assuming the user made a bank transfer to the system using USD.
```

Comment:
The wording for asset is a little confusing here. Do you mean: "'asset' is part of the amount field in a SEP-38 formatted asset..." Or how can we change this wording to make sense?

Reply:
Yeah, unfortunately I couldn't make it more clear. 
Asset if a part of `amount_x` fields (amount_in/out/fee). And it's in a SEP-38 **format**. 
(That's what I was trying to convey at least 😅 )

Reply:
does this make sense then?:  `asset` is part of the `amount_x` field and is in a SEP-38 format.

Reply:
Yes!